kie-server-distributions/pom.xml

```diff
@@ -13,7 +13,7 @@
 
   <properties>
     <version.org.postgresql>42.2.1</version.org.postgresql>
-    <version.org.wildfly>11.0.0.Final</version.org.wildfly>
+    <version.org.wildfly>13.0.0.Final</version.org.wildfly>
```

Comment:
we have in PR queue PR for Wildfly 14.0.1

Wouldn't be better to synchronize on that version?